backend/tests/views/dataset.py

```diff
@@ -92,38 +111,40 @@ def test_datawork_import_for_staff_good(self):
                 "spectros",
             ],
         )
-        self.assertEqual(response.data[0]["name"], "gliderSPAmsDemo (600_400)")
-        self.assertEqual(len(response.data[0]["spectros"]), 1)
-
-    @override_settings(DATASET_IMPORT_FOLDER=IMPORT_FIXTURES / "missing_file")
-    def test_datawork_import_for_staff_missing_file(self):
-        """Dataset view 'datawork_import' returns 'No such file or directory' when missing file"""
-        self.client.login(username="staff", password="osmose29")
-        url = reverse("dataset-datawork-import")
-        response = self.client.get(url)
-        self.assertContains(response, "No such file or directory", status_code=400)
+        self.assertEqual(
+            response.data["result"][0]["name"], "gliderSPAmsDemo (600_400)"
+        )
+        self.assertEqual(len(response.data["result"][0]["spectros"]), 1)
 
     @override_settings(DATASET_IMPORT_FOLDER=IMPORT_FIXTURES / "good")
     def test_datawork_import_for_staff_missing_permissions(self):
         """Dataset view 'datawork_import' returns 'Permission denied' when there is a permission issue"""
         self.client.login(username="staff", password="osmose29")
         url = reverse("dataset-datawork-import")
+        data_send = {"dataset_checked": [{"name": "gliderSPAmsDemo (600_400)"}]}
+        response = self.client.post(
+            url, json.dumps(data_send), content_type="application/json", follow=True
+        )
         original_permissions = settings.DATASET_IMPORT_FOLDER.stat().st_mode
         settings.DATASET_IMPORT_FOLDER.chmod(0o444)  # removing open-folder permission
         try:
-            response = self.client.get(url)
+            response = response = self.client.post(
+                url, json.dumps(data_send), content_type="application/json", follow=True
+            )
         finally:
             settings.DATASET_IMPORT_FOLDER.chmod(original_permissions)
-        self.assertContains(response, "Permission denied", status_code=400)
```

Comment:
Why have you removed status_code checking ? I mean if you changed this to 401 to 403 just change the number not remove the check :wink: (though I guess here you really changed it to 200).